entity-framework/core/what-is-new/ef-core-7.0/whatsnew.md

```diff
@@ -3076,3 +3076,243 @@ LEFT JOIN (
 ) AS [t] ON [b].[Id] = [t].[BlogId]
 ORDER BY [b].[Id], [t].[Title]
 ```
+
+## Faster SaveChanges
+
+In EF7, the performance of <xref:Microsoft.EntityFrameworkCore.DbContext.SaveChanges%2A> and <xref:Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync%2A> has been significantly improved. In some scenarios, saving changes is now four times faster than with EF Core 6.0!
+
+Most of these improvements come from:
+
+- Generating faster SQL
+- Performing fewer roundtrips to the database
```

Comment:
I'd switch the order here :) The fewer roundtrips are more important and universal: the faster SQL is mainly for SQL Server bulk insert, and sometimes for unlocking the fewer roundtrips (through the removal of the transaction).

```suggestion
- Performing fewer roundtrips to the database
- Generating faster SQL
```